gradle/test-retry-gradle-plugin
# pshevche/older gradle no module path

- Add failing tests for a flaky setup preventing retries of failed methods
- Add test checking that setup failures after cleanup failures are handled
- Reword the message for un-retried methods
- Ignore un-retried methods if class-level failure occurred
- Update versions of test frameworks to test against
- Avoid having to espace `\"` in test classes
- Update ASM to 9.5 for Java 21 compatibility
- Enable Dependabot (#183)
- Bump actions/checkout from 2 to 3
- Bump nebula.release from 15.3.1 to 17.2.2
- Bump org.jetbrains:annotations from 23.0.0 to 24.0.1
- Upgrade to 8.1.1
- Bump com.github.johnrengelman.shadow from 7.0.0 to 8.1.1
- Update plugin-publish plugin to 1.2.0 (#193)
- Update copyright year (#194)
- Add Gradle wrapper upgrade plugin and corresponding GitHub Action (#195)
- Bump org.gradle.wrapper-upgrade from 0.11.1 to 0.11.2
- Replace test class inheritence from a spec class with a common abstract spec class
- Fix `ContextualTestExtension` for Spock2
- Add test for handling failed unrolled tests in super class with custom suffix
- Adjust tests to validate classes with parameterized tests not being retried as a whole
- Enable parameterized tests retry if `spock-core-2*` is on the classpath
- Check Spock2 presence on the module path only for Gradle 6.4+
